backend/Testing/CustomAssertions.cs

```diff
@@ -0,0 +1,34 @@
+using System.Diagnostics.CodeAnalysis;
+using FluentAssertions;
+
+namespace Testing;
+
+public static class CustomAssertions
+{
+    public static T ShouldNotBeNull<T>([NotNull] this T? actual, string? customMessage = null)
```

Comment:
I suspect but you'll have to test this, if I write `myVar.ShouldNotBeNull()` I would expect the exception to reference `myVar` however I think right now it'll say `actual` instead. There's some docs about creating custom asserts, it might be as simple as adding an attribute on our methods but I'm not sure. if it's too hard don't worry about it

Reply:
You were right.
So, it turns out that as of quite recently the FluentAssertions 7 prerelease has [NotNull] on appropriate Should methods.
So, I just upgraded instead and shazam everything works.